modules/tinymce/changelog.txt

```diff
@@ -1,3 +1,5 @@
+Version 5.6.1 (TBD)
+    Fixed copying and pasting table columns with a colgroup resulted in invalid HTML #TINY-6684
```

Comment:
Something about the tense is bothering me, but I can't figure out how to improve it. Just commenting in case someone else figures it out, not overly worried though.

Reply:
I had the same thing, I read through the rest of the changelog and there's a mix of past/present/imperative in a few places.

Adding more words to understand the tense makes it a bit easier to follow.

Past
`Fixed an issue where copying and pasting table columns with a colgroup resulted in invalid HTML`

Present
`Fixes a bug where copying and pasting table columns results in invalid HTML`

Imperative(ish)
`This change will fix a bug when copying and pasting table columns with a colgroup, which resulted in invalid HTML`

My suggestion would be to go with past and add "an issue where" to make the tense easier to read
```suggestion
    Fixed an issue where copying and pasting table columns with a colgroup resulted in invalid HTML #TINY-6684
```

Reply:
Added that and reworded a little:
> Fixed an issue where copying and pasting table columns resulted in invalid HTML when using colgroups

Reply:
Yeah that's better nice